app/javascript/components/admin/AdminTable.js

```diff
@@ -639,12 +642,10 @@ class AdminTable extends React.Component {
                 }
                 className="ml-3"
                 disabled={!this.state.actionsEnabled}>
-                {this.state.query.tab !== 'closed' && (
```

Comment:
This _looks_ like a good change, just a bit unexpected in this PR. Was this a bug?

Reply:
It was a remnant of when I first did the AdminTable refactor months ago it looks like. I was surprised to see it. The PatientsTable deals with tabs in the main query, but the AdminTable does not, so I feel it's a safe change. https://github.com/SaraAlert/SaraAlert/blob/7934c39add6ce917befceff91d2e11c21a17a8d7/app/javascript/components/public_health/PatientsTable.js#L78

Reply:
Okay, got it